docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1728,7 +1729,20 @@ hello_name=Hallo {name}! <1> <2>
 
 TIP: An example properties file is generated into the target directory for each message bundle interface automatically. For example, by default if no name is specified for `@MessageBundle` the file `target/qute-i18n-examples/msg.properties` is generated when the application is build via `mvn clean package`. You can use this file as a base for a specific locale. Just rename the file - e.g. `msg_fr.properties`, change the message templates and move it in the `src/main/resources/messages` directory.
 
-Once we have the localized bundles defined we need a way to _select_ a correct bundle.
+A logical line may be spread out across several adjacent normal lines.
```

Comment:
What is a logical line? Can we include a definition for clarity?

Reply:
The term is borrowed from the `java.util.Properties.load(Reader)` javadoc, as metioned below. It's _logical_ because it consists of several "normal" lines.

Reply:
We can include a _definition_ but I believe that the example is better to understand...

Reply:
Right, it's just that this line talks about logical lines as if the reader had any clue about what that is, or maybe I'm dumb :)

Reply:
I get it. How about the new version?